### DISCUSSION TITLE
Input/output delay issue

### DISCUSSION THREAD

**Discussion by DonghanTsai** [bol-edu/HLS-SOC-Discussions#108] [Q&A]
因為有些path的start point可能是input signal
所以當我們做用vivado合成且沒有指定input delay時
該路徑的slack將變成inf
故有可能本來是critical path的路徑並不會在合成時被考量到
所以想請問教授/助教有沒有規定所設定的input delay及output delay？

**Comment by boledulab** [comment]
You may set input delay/output delay as half of the clock.

**Reply by Yanlin900414** [reply]
這是規定clock的一半嗎 ? 還是說只是建議呢 ? 謝謝

**Comment by boledulab** [comment]
你試試不同的input delay， output delay， 因為速度慢，其實沒有什麼影響的。

**Comment by nthuyouwei** [comment]
@DonghanTsai  可以請問你之後是怎麼設定的嗎?在.xdc中
